ajax/libs/timbre/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "timbre",
+    "version": "14.11.25",
+    "filename": "timbre.js",
+    "description": "JavaScript library for objective sound programming",
+    "author": "nao yonamine <mohayonao@gmail.com>",
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/mohayonao/timbre.js.git"
+    },
+    "license": "MIT",
+    "homepage": "https://mohayonao.github.io/timbre.js",
+    "keywords": [
+      "timbre.js",
+      "timbre",
+      "audio",
+      "synth"
+    ],
+    "npmName": "timbre",
+    "npmFileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "timbre.!(node*)*.+(js|map|css)"
```

Comment:
This will only match `timbre.dev.js` - is that intended?

<details><summary>Full package:</summary>
<pre><code>.
├── Gruntfile.js
├── LICENSE
├── README.md
├── build
│   ├── common.jade
│   ├── dev-server.coffee
│   ├── html-builder.coffee
│   ├── package.json
│   ├── test.jade
│   └── timbre-builder.coffee
├── docs.md
│   ├── D-10-GettingStarted.md
│   ├── D-20-PragmaSynth.md
│   ├── D-30-RecordingMode.md
│   ├── D-40-CustomObject.md
│   ├── D-45-ForDevelopers.md
│   ├── D-80-timevalue.md
│   ├── E-beatbox.md
│   ├── E-chord.md
│   ├── E-khoomii.md
│   ├── E-reich.md
│   ├── E-satie.md
│   ├── R-10-BaseObject.md
│   ├── R-20-Number.md
│   ├── R-21-Array.md
│   ├── R-22-Function.md
│   ├── R-23-Boolean.md
│   ├── R-24-Object.md
│   ├── R-31-SynthDef.md
│   ├── R-35-soundbuffer.md
│   ├── R-OscGen.md
│   ├── R-PluckGen.md
│   ├── R-add.md
│   ├── R-audio.md
│   ├── R-biquad.md
│   ├── R-buffer.md
│   ├── R-chorus.md
│   ├── R-clip.md
│   ├── R-comp.md
│   ├── R-delay.md
│   ├── R-dist.md
│   ├── R-div.md
│   ├── R-env.md
│   ├── R-eq.md
│   ├── R-fft.md
│   ├── R-fnoise.md
│   ├── R-gate.md
│   ├── R-ifft.md
│   ├── R-interval.md
│   ├── R-lag.md
│   ├── R-map.md
│   ├── R-max.md
│   ├── R-mediastream.md
│   ├── R-midicps.md
│   ├── R-midiratio.md
│   ├── R-min.md
│   ├── R-mml.md
│   ├── R-mono.md
│   ├── R-mul.md
│   ├── R-ndict.md
│   ├── R-noise.md
│   ├── R-osc.md
│   ├── R-pan.md
│   ├── R-param.md
│   ├── R-phaser.md
│   ├── R-pink.md
│   ├── R-pluck.md
│   ├── R-rec.md
│   ├── R-reverb.md
│   ├── R-schedule.md
│   ├── R-scope.md
│   ├── R-script.md
│   ├── R-selector.md
│   ├── R-spectrum.md
│   ├── R-subtract.md
│   ├── R-tape.md
│   ├── R-task.md
│   ├── R-timeout.md
│   ├── R-waveshaper.md
│   ├── R-zmap.md
│   ├── X-MoogFF.md
│   ├── X-audiojsonp.md
│   ├── X-cosc.md
│   ├── X-keyboard.md
│   ├── X-mouse.md
│   ├── X-mp3_decode.md
│   ├── X-soundfont.md
│   ├── X-webaudioapi.md
│   └── index.md
├── examples
│   ├── audiotest.coffee
│   ├── pico-demo.coffee
│   └── satie.coffee
├── favicon.ico
├── misc
│   ├── audio
│   │   ├── amen.wav
│   │   ├── drum.wav
│   │   ├── drumkit.wav
│   │   ├── guitar.wav
│   │   └── note.mp3
│   ├── audio-jsonp.js
│   ├── css
│   │   ├── bootstrap.css
│   │   ├── codemirror.css
│   │   ├── common.css
│   │   ├── glyphicons-halflings-white.png
│   │   ├── glyphicons-halflings.png
│   │   ├── mocha.css
│   │   └── prettify.css
│   ├── demo.coffee
│   ├── favicon.xcf
│   ├── img
│   │   ├── chrome.png
│   │   ├── firefox.png
│   │   ├── ie.png
│   │   ├── keymap.png
│   │   ├── nodejs.png
│   │   ├── opera.png
│   │   └── safari.png
│   └── js
│       ├── chai.js
│       ├── codemirror.js
│       ├── common.js
│       ├── emacs.js
│       ├── htmlembedded.js
│       ├── htmlmixed.js
│       ├── javascript.js
│       ├── jquery.js
│       ├── mocha.js
│       ├── prettify.js
│       ├── subcollider.js
│       ├── underscore.js
│       └── xml.js
├── package.json
├── src
│   ├── core.js
│   ├── extras
│   │   ├── MoogFF.js
│   │   ├── audio-jsonp.js
│   │   ├── cosc.js
│   │   ├── jsmad.js
│   │   ├── keyboard.js
│   │   ├── min
│   │   │   ├── MoogFF.min.js
│   │   │   ├── audio-jsonp.min.js
│   │   │   ├── cosc.min.js
│   │   │   ├── jsmad.min.js
│   │   │   ├── keyboard.min.js
│   │   │   ├── mouse.min.js
│   │   │   ├── mp3_decode.min.js
│   │   │   ├── pico-binder.min.js
│   │   │   ├── soundfont.min.js
│   │   │   └── webaudioapi.min.js
│   │   ├── mouse.js
│   │   ├── mp3_decode.js
│   │   ├── pico-binder.js
│   │   ├── soundfont.js
│   │   └── webaudioapi.js
│   ├── modules
│   │   ├── biquad.js
│   │   ├── chorus.js
│   │   ├── compressor.js
│   │   ├── decoder.js
│   │   ├── envelope.js
│   │   ├── fft.js
│   │   ├── oscillator.js
│   │   ├── reverb.js
│   │   ├── scissor.js
│   │   └── stereodelay.js
│   └── objects
│       ├── audio.js
│       ├── biquad.js
│       ├── buffer.js
│       ├── chorus.js
│       ├── clip.js
│       ├── comp.js
│       ├── delay.js
│       ├── dist.js
│       ├── div.js
│       ├── env.js
│       ├── eq.js
│       ├── fft.js
│       ├── fnoise.js
│       ├── gate.js
│       ├── ifft.js
│       ├── interval.js
│       ├── lag.js
│       ├── map.js
│       ├── max.js
│       ├── mediastream.js
│       ├── midicps.js
│       ├── midiratio.js
│       ├── min.js
│       ├── mml.js
│       ├── mono.js
│       ├── mul.js
│       ├── ndict.js
│       ├── noise.js
│       ├── osc.js
│       ├── pan.js
│       ├── param.js
│       ├── phaser.js
│       ├── pink.js
│       ├── pluck.js
│       ├── rec.js
│       ├── reverb.js
│       ├── schedule.js
│       ├── scope.js
│       ├── script.js
│       ├── selector.js
│       ├── spectrum.js
│       ├── subtract.js
│       ├── synthdef.js
│       ├── tape.js
│       ├── task.js
│       ├── timeout.js
│       ├── waveshaper.js
│       └── zmap.js
├── test
│   ├── add.js
│   ├── arraywrapper.js
│   ├── booleanwrapper.js
│   ├── clip.js
│   ├── deferred.js
│   ├── div.js
│   ├── events.js
│   ├── functionwrapper.js
│   ├── interval.js
│   ├── max.js
│   ├── min.js
│   ├── mml.js
│   ├── mul.js
│   ├── numberwrapper.js
│   ├── objectwrapper.js
│   ├── osc.js
│   ├── schedule.js
│   ├── subtract.js
│   ├── timbre.debug.js
│   ├── timeout.js
│   └── timevalue.js
├── timbre.as
├── timbre.dev.js
├── timbre.node.js
└── timbre.swf</code></pre>
</details>

Reply:
As I saw in old versions, this one could also fetch map, css files